apps/system/utils/security_level_cmd.c

```diff
@@ -0,0 +1,128 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/ss_info.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <apps/shell/tash.h>
+#include <security/security_ss.h>
+
+/********************************************************************************
+ * Private Functions
+ ********************************************************************************/
+static void tash_set_security_level(int argc, char **args);
+static void tash_get_security_level(int argc, char **args);
+
+const static tash_cmdlist_t security_level_cmds[] = {
+	{"set_sec_level", tash_set_security_level, TASH_EXECMD_SYNC},
+	{"get_sec_level", tash_get_security_level, TASH_EXECMD_SYNC},
+	{NULL, NULL, 0}
+};
+
+static void show_usage(void) 
+{
+	printf("\nUsage : set_sec_level [options]\n");
+	printf("Set your security level high:\n");
+	printf("	set_sec_level high\n");
+	printf("Set your security level low:\n");
+	printf("	set_sec_level low\n");
+}
+
+static void tash_set_security_level(int argc, char **args)
+{
+	char ss_path[7] = "";
+        security_handle hnd;
+	security_data ss_data = {NULL, 0};
+	security_error ret;
+
+	/* Change security_level according to user input */
+	ss_data.length = 2;
+	if (!strncmp(args[1], "high", 5)) {
+		ss_data.data = "1";
+	} else if (!strncmp(args[1], "low", 4)) {
+		ss_data.data = "0";
+	} else if (!strncmp(args[1], "--help", 7)) {
+		show_usage();
+		return;
+	} else {
+		printf("Invalid argument\n");
+		printf("Use [--help] option if you want to know how to use this command\n");
+		return;
+	}
+
+        /* init */
+	ret = security_init(&hnd);
+        if (ret != SECURITY_OK) {
+		printf("[security_init] error ret : %d\n", ret);
+		return;
+	}
+
+	/* Convert secure storage slot index to string path */
+	snprintf(ss_path, 7, "ss/%d", SECURITY_LEVEL_SS_SLOT_INDEX);
```

Comment:
The ss_path has fixed value always, "ss/32". Do you need to get it at runtime?
I think we can use a definition and it will reduce unnecessary operation.